client/src/components/RatingsAndReviews/Sorting.jsx

```diff
@@ -5,7 +5,7 @@ const Sorting = ({sort, setSort}) => {
 
   return (
     <div>
-      <label htmlFor="sortSelect">Sort:</label>
+      <label htmlFor="sortSelect"></label>
```

Comment:
I believe to pass accessibility this needs some text in the label. If you want to hide the label to meet the visual design then use a class I made, pd-visual-hidden